client/components/SignUpForm.js

```diff
@@ -1,11 +1,95 @@
-import React from 'react'
+import React from 'react';
+import {connect} from 'react-redux';
+import {signUpUser, getCurrentUser} from '../actions';
+import axios from 'axios';
+
+class SignUpForm extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      firstName: '',
+      lastName: '',
+      email: '',
+      password: '',
+      cPassword: '',
+      streetAddress:'',
+      city: '',
+      zip:'',
+      state:'',
+    }

```

Comment:
best practice for checking user data on front end before / during submit? and displaying error messages? response from sequelize vs. front-end function in react code or action creator? TBD